src/odemis/gui/cont/tabs/cryo_chamber_tab.py

```diff
@@ -370,15 +382,16 @@ def _reset_project_data(self):
 
     def _load_project_data(self, evt: wx.Event):
```

Comment:
```suggestion
    def _load_project_data(self, evt: wx.Event) -> bool:
```

?

Reply:
I think it's supposed to be `bool`. Please add a docstring to explain what False & True mean.

Then, every place with just `return` should be `return False`.